src/Nri/Ui/Text/V1.elm

```diff
@@ -150,6 +155,34 @@ classString classes =
     String.join " " (List.map (identifierToString namespace) classes)
 
 
+{-| User-generated text.
+-}
+ugMediumBody : List (Html.Styled.Html msg) -> Html.Styled.Html msg
+ugMediumBody =
+    Html.Styled.p
+        [ css
+            [ quizFont
```

Comment:
❓ should `Nri.Fonts` expose a separate `ugFont` even if it's also Georgia?
Not a blocker since that's in `nri-elm-css`, but would like to know what you think.

Reply:
Ooo that's a good idea. 

Also -- I kinda feel like nri-elm-css should merge into this module. Otherwise, updating a font (if we care that everything gets the new font at the same time) means:

- PR -> Review -> Merge -> Release `nri-elm-css`
- PR to make new V2 noredink-ui -> Review -> Merge -> Release `noredink-ui`
- Upgrade to new noredink-ui version & new nri-elm-css version
- Hope that font kerning doesn't mean that we have to change all the hardcoded widths everywhere 😱... so probably use new V2 noredink-ui version & maintain two font styles from nri-eml-css simultaneously?
- PR 🤞  -> Deploy ?

Anyway I'm pro ugFont!

Reply:
Sounds great! Maybe we should bring it up next 🤗& 🐛?